app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 3)) == 20
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(21, 3)
+    assert (max(result) - min(result)) == 0
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(20, 1)
+    assert result == [20]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result = split_integer(20, 3)
+    assert split_integer(20, 1) == [20]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    result = split_integer(5, 8)
+    assert split_integer(5, 8) == [0, 0, 0, 1, 1, 1, 1, 1]
```

Comment:
The assertion `split_integer(5, 8) == [0, 0, 0, 1, 1, 1, 1, 1]` is incorrect because it calls `split_integer` again instead of using the `result` variable. It should be `assert result == [0, 0, 0, 1, 1, 1, 1, 1]`.